src/translate_localization_files.py

```
@@ -955,46 +955,48 @@ async def holistic_review_async(
                     max_tokens=4096,  # Increase tokens to avoid truncation
                     timeout=120.0,
                 )
-                response_text = response.choices[0].message.content.strip()
-                
+                msg_content = response.choices[0].message.content
+                if not msg_content or not msg_content.strip():
+                    logger.error("Holistic review returned empty content.")
+                    raise json.JSONDecodeError("empty response", "", 0)
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)